wp-content/themes/access/controllers/homepage-tout.php

```diff
@@ -69,4 +74,20 @@ public function getStatus() {
       return $status;
     }
   }
+
+  /**
+   * Get the itemtype for structure data
+   *
+   * @return  String  The itemtype
+   */
+
+  public function getItemScope() {
+    if ($this->custom['tout_status_type'] == 'covid-response') {
+      $item_scope = 'SpecialAnnouncement';
+    } else {
+      $item_scope = 'GovernmentService';
```

Comment:
Homepage Touts don't have a fallback schema. Gov't service only applies to programs.